frontend/src/hooks/sails/nft/global.d.ts

```diff
@@ -0,0 +1,62 @@
+import { ActorId } from 'sails-js';
+
+declare global {
```

Comment:
I would better include those types in `lib.ts` file. I know that it's auto generated, but encapsulation to it's business logic is better then globally available types.